habitat_baselines/common/tensor_dict.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+import copy
+import numbers
+from typing import Callable, Dict, Optional, Tuple, Union, overload
+
+import numpy as np
+import torch
+
+TensorLike = Union[torch.Tensor, np.ndarray, numbers.Real]
+DictTree = Dict[str, Union[TensorLike, "DictTree"]]
+TensorIndexType = Union[int, slice, Tuple[Union[int, slice], ...]]
+
+
+def _to_tensor(v: TensorLike) -> torch.Tensor:
```

Comment:
Seems unnecessary. How is this different from [torch.as_tensor](https://pytorch.org/docs/stable/generated/torch.as_tensor.html)

Reply:
Oh yeah, forgot about that.  Thanks!